src/WCSimTrajectory.cc

```diff
@@ -93,7 +96,7 @@ void WCSimTrajectory::ShowTrajectory(std::ostream& os) const
 void WCSimTrajectory::DrawTrajectory(G4int i_mode) const
```

Comment:
Can you `\* i_mode *\` here now, to suppress unused variable compiler warnings? Or maybe better yet, just drop the `G4int` argument to `WCSimTrajectory::DrawTrajectory` altogether?